app/models/post.rb

```diff
@@ -110,7 +110,11 @@ def self.excluding_hidden_content(user)
   end
 
   def self.for_a_stream(max_time, order, user=nil, ignore_blocks=false)
-    scope = self.for_visible_shareable_sql(max_time, order).
+    self.for_a_ranged_stream(Date.parse('01-01-2008'), max_time, order, user, ignore_blocks)
```

Comment:
- Style/RedundantSelf: Redundant `self` detected.
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.